### DISCUSSION TITLE
The mesh is only in ram

### DISCUSSION THREAD

**Discussion by cihanaygar** [Jyers/Marlin#410] [Q&A] [answered]
Hello. 3d printer for hobby. I do not know much about 3d printers. There is v4.2.2 motherboard. Version UBL Bltouch 15x15 installed. I get an error at step 157 when creating a new mesh. I am getting an error that says  "The mesh is ONLY in RAM! Use Save Mesh to save". I do not have much experience. can you help me?

![1](https://user-images.githubusercontent.com/82159111/114050171-b5f13e00-9894-11eb-835d-ec5505d17727.jpeg)
![2](https://user-images.githubusercontent.com/82159111/114050185-b984c500-9894-11eb-94c6-d9e01d4a05c6.jpeg)

**Comment by Cal-LF** [comment, marked as answer]
It probes only what it can reach, the rest is inferred. It is working properly.

**Reply by cihanaygar** [reply]
Thank you very much for your help.
